rfc/src/rfcs/0007-global-conditions.md

```diff
@@ -0,0 +1,97 @@
+- **Feature Name:** `global-conditions`
+- **Feature Request Issue:** *N/A*
+- **RFC PR:** *Link to original PR*
+- **Status:** *Under Review*
+- **Version:** 0
+- **Proof-of-concept:** TBD
+
+## Summary
+
+A new section in Kani's output to summarize the status of hyperproperties.
+
+## User Impact
+
+The addition of new options that affect the overall verification result depending on certain property attributes demands some consideration.
+In particular, the addition of a new option to fail verification if there are uncoverable (i.e., unsatisfiable or unreachable) `cover` properties (requested in [#2299](https://github.com/model-checking/kani/issues/2299)) is posing new challenges to our current architecture and UI.
+
+Hyperproperties are properties that depend on attributes of other properties.
```

Comment:
Is this a standard definition of hyperproperties? The definition I'm aware of is that while properties refer to traces, hyperproperties refer to sets of traces:
https://en.wikipedia.org/wiki/Hyperproperty
The definition used here doesn't seem to match with that.